content/en/storage-providers/setup/prerequisites.md

```diff
@@ -46,6 +46,10 @@ export FIL_PROOFS_PARAMETER_CACHE=/path/to/folder/in/fast/disk
 export FIL_PROOFS_PARENT_CACHE=/path/to/folder/in/fast/disk2
 ```
 
+Make sure the `ListenAddress` has [remote access enabled]({{< relref "reference/basics/overview" >}}). Instructions on how to obtain a token are [available here]({{< relref "api-access" >}}).
```

Comment:
Using more specific redirects to avoid confusion for new users.

```suggestion
Make sure the `ListenAddress` has [remote access enabled]({{< relref "reference/basics/api-access/#enable-remote-api-access" >}}). Instructions on how to obtain a token are [available here]({{< relref "reference/basics/api-access/#api-tokens" >}}).
```